chart/compass/values.yaml

```diff
@@ -132,13 +135,15 @@ global:
     caCertificate: ""
 
   system_broker:
-    port: 5001
-    tls:
-      host: compass-gateway
     enabled: true
+    port: 5001
+    prefix: /broker
+    forwardHeaders: Authorization
+    unauthorizedString: "insufficient scopes provided"
```

Comment:
Why externalize this error message?

Reply:
We externalized this error to make it configurable since the default value is an empty string as this property is something specific to different brokers.